rustup-init.sh

```diff
@@ -15,14 +15,13 @@
 
 set -u
 
-if [ -z "$RUSTUP_UPDATE_ROOT" ]; then
-    RUSTUP_UPDATE_ROOT="https://static.rust-lang.org/rustup"
-fi
+RUSTUP_UPDATE_ROOT=${RUSTUP_UPDATE_ROOT:-"https://static.rust-lang.org/rustup"}
 
```

Comment:
This change fixes Undefined Variables error in `RUSTUP_UPDATE_ROOT` because of setting `set -u`.